lib/media/streaming_engine.js

```diff
@@ -157,6 +157,9 @@ shaka.media.StreamingEngine = class {
     /** @private {?shaka.util.Timer} */
     this.updateLiveSeekableRangeTime_ = new shaka.util.Timer(() => {
```

Comment:
Could you please also correct this from `Time_` to `Timer_`?  It confuses me every time I read it.

Reply:
Done!